ui/app/helpers/confirm-transaction/util.js

```diff
@@ -137,3 +138,11 @@ export function convertTokenToFiat ({
 export function hasUnconfirmedTransactions (state) {
   return unconfirmedTransactionsCountSelector(state) > 0
 }
+
+export function roundExponential (value) {
+  const PRECISION = 4
+  const valueBN = new BigNumber(value)
```

Comment:
nit: can `valueBN` be renamed so that it's clear it is not a `BN.js` object (which is used much more in the old-ui but behaves differently, especially with respect to decimals)

Reply:
> so that it's clear it is not a `BN.js` object

It's not?

Reply:
> It's not?

I suppose my issue with this is that, at least in the old-ui, and probably in some utilities still in the new ui, there are bunch of variables with `BN` in their name to indicate it is a `BN.js` object. So, for instance, if I was to grep for all BNs, this would have been confusing. Or, at least, reading this code and then reading. Maybe that is a minor concern, and I'd just like us to not pollute something of a convention in the codebase (e.g. see metamask-extension/ui/app/util.js)

But you are right that reading these few lines of code it should be clear.

Reply:
`BN.js` being different from `BigNumber`?